compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -1030,6 +1031,20 @@ void OperationExporter::visit(luci::CircleSpaceToBatchND *node)
   gd._operators.push_back(op_offset);
 }
 
+void OperationExporter::visit(luci::CircleSpaceToDepth *node)
+{
+  uint32_t op_idx = md.registerBuiltinOpcode(circle::BuiltinOperator_SPACE_TO_BATCH_ND);
```

Comment:
maybe not changed?

Reply:
Sorry about this, I fixed it. Thank you for catching it.